README.md

```diff
@@ -45,11 +45,21 @@
 
   To run this program simply clone the repo from GitHub using the following command, 
   
+  ```bash
+  git clone git@github.com:Thenlie/readme-generator.git || git clone https://github.com/Thenlie/readme-generator.git
```

Comment:
I think I would prefer something like:
---
Use the steps below to clone the repository and get the project running on your local machine.

1. Navigate to the develop branch of the repository and click the green "Code" button. Then copy the repository URL with the method of your choosing. We suggest SSH which can be setup be following [these docs](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).

2. Go to your terminal program of choice and navigate the the folder you want the project to live in. Then run the command below to clone the repository:

```sh
git clone <repository_url>
```
---
This let's the user know that we are showing steps for SSH, but still gives them the flexibility to choose HTTPS.